extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/music/GoogleMusicExporter.java

```diff
@@ -280,7 +280,8 @@ private MusicPlaylistItem convertPlaylistItem(
                 release.getIcpn(),
                 release.getTitle(),
                 createMusicGroups(release.getArtists())),
-            createMusicGroups(track.getArtists())),
+            createMusicGroups(track.getArtists()), false),
+        // TODO: Add explicit flag to GooglePlaylistItem
```

Comment:
is this a nice-to-have at some point or pre-launch? (should we use the other TODO-format we were using?)

Reply:
It's pre-launch. Updated the TODO format.